### PR TITLE
Fix spacing for Add Display flow [stage-1]

### PR DESCRIPTION
**Pull request by alex-deaconu**

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@Rise-Vision/apps please review. Thanks!

File: web/scripts/config/stage.js [hunk 1]
```diff
@@ -30,7 +30,6 @@
     .value('VIEWER_URL', 'http://stage-test-dot-rvaviewer-test.appspot.com')
     .value('ALERTS_WS_URL',
       'https://rvacore-test.appspot.com/alerts/cap')
-    .value('SEGMENT_API_KEY', null)
```
Comment (ezequielc) [inline web/scripts/config/stage.js]: @alex-deaconu Should we revert this to `null` before merging? Thanks

Comment (alex-deaconu) [reply web/scripts/config/stage.js]: @ezequielc good point, I will revert as part of the CH update tag commit. Thanks for the reminder!

**Comment by alex-deaconu** [comment]
@ezequielc updated the Image tag to use the responsive class, thanks for the tip.

Last commit also includes reverting the config change, re-naming the variable as per typo and updating the CH tag.

Thanks.

**Comment by ezequielc** [comment]
👍

**Pull request merged**